api-tests/core/strapi/document-service/publish.test.api.ts

```diff
@@ -101,21 +65,21 @@ describe('Document Service', () => {
       })
     );
 
-    it.skip(
+    it(
       'publishes one locale of a document with multiple locales when locale is string',
       testInTransaction(async () => {
         const results = await strapi.documents.publish(ARTICLE_UID, 'Article1', {
-          locales: ['en'],
+          locales: ['en'], // Publish only english
```

Comment:
Should this work the same with locales: 'en' then ? based on the test description

Reply:
yes it should !